src/internet_identity/src/http.rs

```diff
@@ -53,6 +53,28 @@ pub fn http_request(req: HttpRequest) -> HttpResponse {
                 },
             }
         }
+
+        // the /version endpoint returns information about the version.
+        // Technically this should only accept requests with 'accept: text/plain' but we don't
+        // check headers anywhere else so neither do we here.
+        "/version" => {
+            let version_info =
+                crate::VERSION_INFO.with(|version_info| version_info.borrow().clone());
```

Comment:
I'm not super happy with this `.clone()` but the `body` of HttpResponse either needs a `'static` (which is not ideal here) or for the data to be owned. I have no idea why the type is set up as is right now and will need to do some digging (independently of the PR).

Reply:
I don't think cloning such a small value is a problem. But yeah, we should definitely find out and document the reasoning behind the typing.